shared/login/relogin/container.tsx

```diff
@@ -56,15 +61,16 @@ type State = {
 }
 
 type Props = {
-  users: Array<string>
-  onForgotPassword: () => void
-  onSignup: () => void
-  onSomeoneElse: () => void
   bannerError: string
   inputError: string
-  selectedUser: string
+  loggedInMap: I.Map<string, boolean>
```

Comment:
we try and keep immutable out of props for components

Reply:
Why is that?

Reply:
because we have a separation of concerns. we can discuss irl if you wish. also for consistencies sake

Reply:
sure, I gotta go but will poke you tomorrow. Already made the change - I don't feel strongly about it, just was curious.